frontend/src/app/page.tsx

```
@@ -1,12 +1,171 @@
+
+    return false;
+  }
+
+  async function fetchTrendingContent(): Promise<boolean> {
+    try {
+      const trendingResponse = await axios.get(
+        `${apiURL}/content/feed/trending`,
+        { timeout: 5000 }
+      );
+
+      if (trendingResponse.data && trendingResponse.data.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)